crates/compiler/can/src/def.rs

```diff
@@ -2507,6 +2536,27 @@ pub fn can_defs_with_return<'a>(
     (loc_expr.value, output)
 }
 
+pub fn report_unused_imports(
+    imports_introduced: Vec<IntroducedImport>,
+    references: &References,
+    env: &mut Env<'_>,
+    scope: &mut Scope,
+) {
+    for import in imports_introduced {
+        if references.has_module_lookup(import.module_id) {
+            for (symbol, region) in import.exposed_symbols {
+                if !references.has_unqualified_type_or_value_lookup(symbol)
+                    && !scope.abilities_store.is_specialization_name(symbol)
+                {
```

Comment:
could this be replaced with `references.has_*_lookup`?

Reply:
@ayazhafiz I'm not sure what you meant here. This is already using a `References` method. Are you talking about the `scope.abilities_store.is_specialization_name` check?

Reply:
yes, sorry, I mean that I am a bit surprised that `scope.abilities_store.is_specialization_name` is not synonymous with a reference for that symbol already existing - although the lack of that may be my fault. I wonder if there's an easy way to combine them into one method.